public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,320 @@
+openapi: 3.0.0
+info:
+  title: User Information
+  description: >-
+    Access to user information is controlled through scopes. Each access token (OAuth 2.0 Bearer Token) is associated with a set of scopes at login. When a request is made for user information, only information belonging to the provided scopes is returned. The information is returned in the form of claims, which sometimes are simple fields and sometimes objects that contain further fields.
+    Here is a list of supported scopes and the claims they contain. The details of each claim, including any contained fields, is documented further down.
+    * 'profile': given_name, middle_name, family_name, birthdate
+    * 'address': address
+    * 'email': email
+    * 'openid:hmrc-enrolments': hmrc_enrolments
+    * 'openid:government-gateway': government_gateway
+    * 'openid:mdtp': mdtp
+    * 'openid:gov-uk-identifiers': uk_gov_nino
+  contact: {}
+  version: '1.0'
+servers:
+- url: https://api.service.hmrc.gov.uk/
+  variables: {}
```

Comment:
At root level, we need to insert a `securitySchemes` property as per this example: <https://github.com/hmrc/open-api-example-microservice/blob/main/resources/public/api/conf/1.0/application.yaml#L42>.

Ensure the scopes match those provided in the API definition: https://github.com/hmrc/openid-connect-userinfo/blob/main/app/views/definition.scala.txt.

Finally, each endpoint needs a `security` property declaring to which scopes it is restricted, as shown on these lines:
- https://github.com/hmrc/open-api-example-microservice/blob/main/resources/public/api/conf/1.0/application.yaml#L91
- https://github.com/hmrc/open-api-example-microservice/blob/main/resources/public/api/conf/1.0/application.yaml#L137
- https://github.com/hmrc/open-api-example-microservice/blob/main/resources/public/api/conf/1.0/application.yaml#L184

You may need to refer to the RAML or consult someone familiar with this service to determine what those restrictions are.